sorc/ncep_post.fd/SURFCE.f

```diff
@@ -1,65 +1,61 @@
-      SUBROUTINE SURFCE
-!$$$  SUBPROGRAM DOCUMENTATION BLOCK
-!                .      .    .     
-! SUBPROGRAM:    SURFCE      POST SURFACE BASED FIELDS
-!   PRGRMMR: TREADON         ORG: W/NP2      DATE: 92-12-21       
-!     
-! ABSTRACT:
-!     THIS ROUTINE POSTS SURFACE BASED FIELDS.
-!   .     
-!     
-! PROGRAM HISTORY LOG:
-!   92-12-21  RUSS TREADON
-!   94-08-04  MICHAEL BALDWIN - ADDED OUTPUT OF SFC FLUXES OF
-!                               SENS AND LATENT HEAT AND THETA AT Z0
-!   94-11-04  MICHAEL BALDWIN - ADDED INSTANTANEOUS PRECIP TYPE
-!   96-03-19  MICHAEL BALDWIN - CHANGE SOIL PARAMETERS
-!   96-09-25  MICHAEL BALDWIN - ADDED SNOW RATIO FROM EXPLICIT SCHEME
-!   96-10-17  MICHAEL BALDWIN - CHANGED SFCEVP,POTEVP TO ACCUM.  TOOK
-!                               OUT -PTRACE FOR ACSNOW,SSROFF,BGROFF.
-!   97-04-23  MICHAEL BALDWIN - TOOK OUT -PTRACE FOR ALL PRECIP FIELDS
-!   98-06-12  T BLACK         - CONVERSION FROM 1-D TO 2-D
-!   98-07-17  MIKE BALDWIN - REMOVED LABL84
-!   98-08-18  MIKE BALDWIN - COMPUTE RH OVER ICE
-!   98-12-22  MIKE BALDWIN - BACK OUT RH OVER ICE
-!   00-01-04  JIM TUCCILLO - MPI VERSION
-!   01-10-22  H CHUANG - MODIFIED TO PROCESS HYBRID MODEL OUTPUT
-!   02-06-11  MIKE BALDWIN - WRF VERSION ASSUMING ALL ACCUM VARS
-!                            HAVE BUCKETS THAT FILL FROM T=00H ON
-!   02-08-28  H CHUANG - COMPUTE FIELDS AT SHELTER LEVELS FOR WRF
-!   04-12-09  H CHUANG - ADD ADDITIONAL LSM FIELDS
-!   05-07-07  BINBIN ZHOU - ADD RSM MODEL
-!   05-08-24  GEOFF MANIKIN - ADDED DOMINANT PRECIP TYPE
-!   11-02-06  JUN WANG  - ADDED GRIB2 OPTION
-!   13-08-05  S Moorthi - Eliminate unnecessary arrays (reduce memory)
-!                         and some cosmetic changes
-!   14-02-26  S Moorthi - threading datapd assignment
-!   14-11-26  S Moorthi - cleanup and some bug fix (may be?)
-!   20-03-25  J MENG    - remove grib1
-!     
-! USAGE:    CALL SURFCE
-!   INPUT ARGUMENT LIST:
-!
-!   OUTPUT ARGUMENT LIST: 
-!     
-!   OUTPUT FILES:
-!     NONE
-!     
-!   SUBPROGRAMS CALLED:
-!     UTILITIES:
-!       BOUND    - ENFORCE LOWER AND UPPER LIMITS ON ARRAY ELEMENTS.
-!       DEWPOINT - COMPUTE DEWPOINT TEMPERATURE.
-!       CALDRG   - COMPUTE SURFACE LAYER DRAG COEFFICENT
-!       CALTAU   - COMPUTE SURFACE LAYER U AND V WIND STRESSES.
+!> @file
 !
-!     LIBRARY:
-!       COMMON   - CTLBLK
-!                  RQSTFLD
-!     
-!   ATTRIBUTES:
```

Comment:
I've noticed in a couple routines that the Attributes Language and Machine don't translate over.  Is that an issue or not really necesary?

Reply:
I usually take those out, as they don't seem to be relevant. The language is evident from the code and the filename, and the machine it was developed on is not relevant any longer.